website/docs/r/vapp_org_network.html.markdown

```diff
@@ -37,6 +44,12 @@ The following arguments are supported:
 * `org_network_name` - (Optional; *v2.7+*) An Org network name to which vApp network is connected. If not configured, then an isolated network is created.
 * `is_fenced` (Optional) Fencing allows identical virtual machines in different vApp networks connect to organization VDC networks that are accessed in this vApp. Default is false.
 * `retain_ip_mac_enabled` - (Optional) Specifies whether the network resources such as IP/MAC of router will be retained across deployments. Configurable when `is_fenced` is true.
+* `reboot_vapp_on_destroy` - (Optional; *v3.9+*) **VCD 10.4.1+** API **prohibits removal of vApp
+  network from a powered on vApp**. This field can be used to power off vApp during vApp network
+  removal. It will power on the vApp (if it was so) after removing network. (default `false`)
+  **Note.** It only affects *destroy* operation for the resource and will never power cycle vApp
```

Comment:
Apparently, I have an issue with using "destroy" in the context of a single resource :) Shall we standardize on "delete"? I'd like us to use the most correct term here to avoid any confusion.

Reply:
Changed `destroy` to `delete` apart from the field name, which we discuss in separate thread https://github.com/vmware/terraform-provider-vcd/pull/1004#discussion_r1118488408